CHANGELOG.rst

```diff
@@ -2,14 +2,18 @@
 Change history
 ==============
 
-3.5.0 (TBD)
------------
+3.5.0 (2025-12-01)
+------------------
+
+**New features**
+
+* [:open-api-framework:`176`] Upgrade ``mozilla-django-oidc-db`` to 1.1.1
```

Comment:
I think i'd remove this part (since it's also mentioned under `Maintenance` and just have the warning sit at the top. You could move `**New features**` to right above OTel